tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/QueueControlTest.java

```diff
@@ -2618,6 +2618,60 @@ public void testMoveMessage() throws Exception {
       session.deleteQueue(otherQueue);
    }
 
+   @TestTemplate
```

Comment:
For adding a feature like this you should ideally write a test that covers more of the protocols and messages (normal or large) as well as times when paging is happening vs not as there have been a number of cases lately where a breakage or misbehavior has been found when an AMQP or a Large message was involved in internal management code such as this. 

Refer to test: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/plugin/BeforeSendPluginTest.java for some inspiration